AirCasting/SessionViews/ABMeasurementsView.swift

```diff
@@ -3,27 +3,26 @@
 
 import SwiftUI
 import AirCastingStyling
-import CoreLocation
-import CoreData
 
 enum MeasurementPresentationStyle {
     case showValues
     case hideValues
 }
 
 struct ABMeasurementsView: View {
+    
+    @StateObject private var viewModel = ViewModel()
```

Comment:
It's best when viewmodels are being passed in instead of created by the view layer itself. But as currently we have no good way of injecting dependencies I'm not sure there is a difference.

Reply:
I had to do it anyways after marking `measurementStreamStorage: MeasurementStreamStorage?` as private.